packages/frontend-2/pages/workspaces/[slug]/dashboards/[id].vue

```diff
@@ -98,15 +97,12 @@ const editDialogOpen = ref(false)
 
 const workspace = computed(() => result.value?.dashboard?.workspace)
 const dashboard = computed(() => result.value?.dashboard)
-const dashboardUrl = computed(() => {
-  return urlToken
-    ? `${dashboardsOrigin}/view/${id}?token=${urlToken}&isEmbed=true&theme=${
-        isDarkTheme.value ? 'dark' : 'light'
-      }`
-    : `${dashboardsOrigin}/dashboards/${id}?token=${
-        effectiveAuthToken.value
-      }&isEmbed=true&theme=${isDarkTheme.value ? 'dark' : 'light'}`
-})
+const dashboardUrl = computed(
+  () =>
+    `${dashboardsOrigin}/dashboards/${id}?token=${
```

Comment:
y not use the URL() class and URL().searchParams, easier and less brittle than stitching together strings, ensuring proper formatting and encoding etc.

Reply:
if u need a RelativeURL, there's RelativeURL in shared